content/zh-TW/api_references/data_channel_format.md

```diff
@@ -226,6 +242,31 @@ JSON 格式：
 PWM的值和期間只能為 0 至 1000 之間。
 
 CSV 格式：
+
 ```
 dataChannelId,timestamp,{Value},{Period}
 ```
+
+## 類比
+JSON 格式：
+
+```
+{
+ "datapoints":[
+      {
+         "dataChnId":"dataChanId",
+         "values":{
+            "value":"{Integer value}"
+         }
+      }
+   ]
+}
+
+```
+
+CSV 格式：
+
+```
+dataChannelId,timestamp,{Integer value}
+```
```

Comment:
Hi @appleboy, because the format is the same as integer, it should be clear for users even if we only provide the format description. How do you think?